pypit/arutils.py

```diff
@@ -190,15 +192,17 @@ def find_peaks(yval, siglev=10.):
       Sigma level for detection
     """
     # Calculate RMS
-    yclipped = sig_clip(yval)
+    from pypit import arcyarc
+    yclipped = sigma_clip(yval)
     rms = np.std(yclipped)
     #
-    tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev) 
+    #tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev)
+    debugger.set_trace()  # FIX THE ROW ABOVE
```

Comment:
mid_row isn’t defined.

I’m not sure what to do..

X

> On Jun 9, 2016, at 10:46 AM, Ryan Cooke <notifications@github.com> wrote:
> 
> In pypit/arutils.py <https://github.com/PYPIT/PYPIT/pull/84#discussion_r66487873>:
> 
> >      rms = np.std(yclipped)
> >      #
> > -    tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev) 
> > +    #tpixt, num = arcyarc.detections_sigma(yval,np.array([rms]*mid_row.shape[0]),np.zeros(yval.shape[0],dtype=np.int),siglev/2.0,siglev)
> > +    debugger.set_trace()  # FIX THE ROW ABOVE
> Should something be returned from this function after rms is calculated? If there's a bug here, perhaps this should be filed as an issue --- to remind me to look into it :-)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/PYPIT/PYPIT/pull/84/files/2565928f100bdbc4d475b9a9ff8f53779e4b683e#r66487873>, or mute the thread <https://github.com/notifications/unsubscribe/AD0BjXR8_4soT32umLPkF1dDE4x-T7nKks5qKFGNgaJpZM4IxqGg>.
> 
> !DSPAM:826,5759ab93154201579910136!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/pyigm/pyigm
https://github.com/linetools/linetools

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)